saleor/graphql/account/mutations/authentication/request_password_reset.py

```diff
@@ -72,58 +69,31 @@ def clean_user(cls, email, redirect_url, info: ResolveInfo):
             )
 
         user = retrieve_user_by_email(email)
-        if not user:
-            raise ValidationError(
-                {
-                    "email": ValidationError(
-                        "User with this email doesn't exist",
-                        code=AccountErrorCode.NOT_FOUND.value,
-                    )
-                }
-            )
-
-        if not user.is_active:
```

Comment:
Should we allow reset passwords for non-active users?

Reply:
Good catch, fixed along with the test that was actually false positive